geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -17,55 +17,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-The Redis API for <%=vars.product_name%> allows <%=vars.product_name%> to function as a drop-in replacement for a
+The <%=vars.product_name%> API for Compatibility with Redis allows <%=vars.product_name%> to function as a drop-in replacement for a
```

Comment:
The <%=vars.product_name%> APIs compatible with Redis allows <%=vars.product_name%> to function as a drop-in replacement for a